src/components/button/Button.styles.js

```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+    buttonEnabled: {
```

Comment:
This is general styles for button. Try to separate `buttonContainer` and then you can do `buttonDisable` and `buttonEnable` styles.

Reply:
I've added a general `buttonContainer` style